content/docs/custom-plugins/authenticated-proxy/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Configuring Authenticated Proxy for Custom Plugins
+lastUpdated: '2023-02-03T10:27:00.0Z'
+description: How to configure a custom proxy to identify the Backstage user for you custom plugin.
+---
+
+## Introduction
+
+Roadie provides a custom proxy service that can be configured to proxy through an access token minted by our own authentication system that can be validated by your third party service.
+
+
+## Step 1. Create an authenticated proxy configuration in Roadie
+
+You can navigate to h<gatsbyhack>tt</gatsbyhack>ps://your-company.roadie.so/administration/settings/authenticatedProxy. Click add item to add a new proxy.
```

Comment:
Not sure this will play well with the techdocs. Can you try something else?

Reply:
What aspect will not play well?

Reply:
What do I need to do to make it play well?

Reply:
<gatsbyhack>

Reply:
You could make this a link?

Reply:
Or just plain txt

Reply:
I made a change, Is that what you mean?